src/Groups/matrices/matrix_manipulation.jl

```diff
@@ -254,5 +253,5 @@ end
 
 # evaluation of the form x into the vectors v and u
 Base.:*(v::AbstractAlgebra.Generic.FreeModuleElem{T},x::MatrixGroupElem{T},u::AbstractAlgebra.Generic.FreeModuleElem{T}) where T <: RingElem = (v.v*matrix(x)*transpose(u.v))[1]
-
-map(f::Function, v::AbstractAlgebra.Generic.FreeModuleElem{T}) where T <: RingElem = v.parent(map(f,v.v))
+#
+#map(f::Function, v::AbstractAlgebra.Generic.FreeModuleElem{T}) where T <: RingElem = v.parent(map(f,v.v))
```

Comment:
Besides being type piracy, it also probably isn't the correct idea to use `map` here. Maybe use `map_entries`? And move it to AA?

Reply:
OK I checked what this is used for, and as far as I can tell it is used exclusively to apply a Frobenius endomorphism to a `FreeModuleElem`. For that it would make more sense to have a dedicated function specialized (and optimized) for that case.

There is more code in `src/Groups/matrices` where this applies, just that there matrices are transformed instead of vectors. Prototypical:
```julia
function conjugate_transpose(x::MatElem{T}) where T <: FinFieldElem
   @req iseven(degree(base_ring(x))) "The base ring must have even degree"
   e = div(degree(base_ring(x)),2)
   return transpose(map(y -> frobenius(y,e),x))
end
```
This could be implemented much more efficiently (with less allocation overhead) by combing transpose and Frobenius into one step. 

And then one step further, we can look at where it is applied:
```
      m = x^-1*gram_matrix(f)*conjugate_transpose(x^-1)
```
I think we could do better there, too...